### DISCUSSION TITLE
Invisible crossing=traffic_signals

### DISCUSSION THREAD

**Discussion by hungerburg** [openstreetmap/iD#8866] [Q&A] [answered]
This was issue #8516 - the iD editor does not show nodes with that tag anything special. Yet they are. A similar issue came up in the reddit QA of the new maintainer. What is wrong with this tag? Why must it be suppressed by iD?

**Comment by tyrasd** [comment, marked as answer]
Hi!

This missing preset was finally added a few weeks ago in https://github.com/openstreetmap/id-tagging-schema/pull/192 and should be available with the next release of the tagging presets (which you can expect in about one or two weeks).
